common/src/main/scala/com/gu/anghammarad/models/models.scala

```diff
@@ -46,7 +47,8 @@ case class Notification(
   actions: List[Action],
   target: List[Target],
   channel: RequestedChannel,
-  sourceSystem: String
+  sourceSystem: String,
+  threadKey: Option[String] // only used for Hangouts messages
```

Comment:
This `Notification` case class is the contract between Anghammarad and users. Adding the additional field as an `Option` should mean this change is backwards compatible.

Reply:
Minor: we could default this to `None`?

Reply:
Ah yes! I hadn't set a default value so I could see all the compiler warnings. Can safely set one now though.